### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by randomstuff**

### CODE REVIEW

File: core/nullDC.cpp [hunk 1]
```diff
@@ -196,7 +197,6 @@ int dc_init(int argc,wchar* argv[])
 		printf("Using Interpreter\n");
 	}
 	
-	  void InitAudio();
```
Comment (skmp) [inline core/nullDC.cpp]: I'm just too damned lazy sometimes -.-

**Comment by skmp** [comment]
I'm favor apart from the verify changes -- the code should be changed to be like rbi != 0 (and the ==false left in place). Maybe i'm nitpicking. Can you rebase to latest master?

**Comment by randomstuff** [comment]
Yes, your solution for `verify` is better…

**Comment by randomstuff** [comment]
Rebased and fixed the `verify` macro usage.

**Comment by skmp** [comment]
Does this also address #718 ?

**Pull request merged**